README.md

```diff
@@ -36,7 +36,7 @@ An attractive, custom analytics dashboard that's ready to be shown to your team
 
 ## The Templates
 
-These layout templates are composed of a minimal set of [Bootstrap v3.2](http://getbootstrap.com/) custom styles. They cover the most common use cases and layout configurations we've encountered so far.
+These layout templates are composed of a minimal set of custom styles. They cover the most common use cases and layout configurations we've encountered so far.
 
 * [Layouts](http://keen.github.io/dashboards/layouts/) for pre-built, responsive dashboard views
 * [Examples](http://keen.github.io/dashboards/examples/) for specific domains, data models and popular integrations
```

Comment:
please update dashboards/assets/js/keen-analytics.js file:
https://d26b395fwzu5fz.cloudfront.net/keen-tracking-1.1.4.min.js
change to the current version from jsdeliver